src/interface/CharacterParses.tsx

```diff
@@ -397,7 +406,16 @@ class CharacterParses extends Component<CharacterParsesProps, CharacterParsesSta
           });
           return;
         }
-        // hidden isn't a value on the parse response
+
+        // WCL responds with {hidden:true} when the logs are hidden.
+        if (rawParses?.hidden) {
```

Comment:
Typecheck fails on `Property 'hidden' does not exist on type 'WCLParsesResponse'.`

Im not sure how to properly work around this, any suggestions?